docs/development/release.md

```diff
@@ -260,6 +260,13 @@ Finally, you need to reproduce the version update changes back to master.
 
 If this is the latest version, you can simply create a PR to merge the minor version base branch back to master.
 
+Do not merge `master` back into the release branch **until**:
```

Comment:
Should we **ever** be merging `master` into the release branch? The whole point of the release branch is that it represents the state of the code at the time of that release. Merging `master` back into the release branch pollutes that so you can't cut new releases out of the release branch without including changes from `master` that weren't in the original release.

Does that make sense?

Reply:
@markdboyd  partly.  

We don't want to merge master into release *as soon as* there's a breaking change. 

So for example:

---
9.2.0 gets released and a branch is created
Changes go in that only require 9.2.1.   In that case, we'd merge master to the 9.2.x branch, and then cut a release PR for 9.2.1
---

vs

---
9.2.0 gets released and a branch is created

Changes go in that only require 9.3.0.   In that case, a new release maintenance branch would get created. 
---

It might be worth having another brown bag re: our current release setup....

Reply:
Got it. I definitely did not recognize or appreciate that distinction. It's safe to merge `master` back to `9.2.x` if we're making a minor release. Also merging `master` to `release-9.2.x` is not the same as merging `master` back to `release-9.2.0`

Reply:
Thanks for the conversation here 😄. A team discussion/brownbag sounds like a good idea to surface questions people might have.